docs/how-to-guides/develop-a-prompty/prompty-output-format.md

```diff
@@ -22,7 +22,6 @@ model:
   api: chat
   configuration:
     type: azure_openai
-    connection: <connection_name>
```

Comment:
should add a real connection name like what other places in this page do?

Reply:
fix it, add connection name here